app/messaging/message-receiver.js

```diff
@@ -1,6 +1,6 @@
 const { ReceiveMode } = require('@azure/service-bus')
 const MessageBase = require('./message-base')
-const { logTraceMessage } = require('../app-insights')
+const { logTraceMessage, trackException } = require('../app-insights')
```

Comment:
Can we rename the method `trackTrace `to keep it consistent with the `trackException`